materials/knitr-examp.Rmd

```diff
@@ -1,8 +1,8 @@
 ---
 title: "Portal population dynamics"
-author: "Ethan White"
+author: "Ethan White and David J. Harris"
```

Comment:
Note: this isn't intended to signify the author of the lesson, but to be used to demo the `author` field when teaching the lesson. It could be replaced with `Instructor's Name` or something like that to help indicate this.